docs/configuration_file.md

```diff
@@ -27,6 +27,8 @@ Some options are also mandatory.
 
 * `verbose` (*optional*, `boolean`) — enable verbose logging
 
+* `openshift_required_version` (*optional*, `string`) — required version to run against (adjusts build template as appropriate)
```

Comment:
It would be best not to need such option at all but there doesn't seem to be API endpoint for obtaining the server version.